src/models.jl

```diff
@@ -0,0 +1,156 @@
+"""
+    Models for random hypergraphs building:
+        * random model
+        * k-uniform model
+        * d-uniform model
+        * preferential-attachment
+"""
+
+
+"""
+    random_model(nVertices, nEdges)
+
+Generate a *random* hypergraph without any structural constraints.
+
+**The Algorithm**
+Given two integer parameters *nVertices* and *nEdges* (the number of nodes and hyperedges, respectively),
+the algorithm computes - for each hyperedge *he={1,...,m}* -
+a random number *s ϵ [1, n]* (i.e. the hyperedge size).
+Then, the algorithm selects uniformly at random *s* vertices from *V* to be added in *he*.
+"""
+function random_model(nVertices::Int, nEdges::Int)
+    mx = Matrix{Union{Nothing,Bool}}(nothing, nVertices,nEdges)
+    for e in 1:size(mx,2)
+        nv = rand(1:size(mx,1))
+        mx[sample(1:size(mx,1), nv;replace=false), e] .= true
+    end
+
+    h = Hypergraph(mx)
+    if all(length.(h.v2he) .> 0)
```

Comment:
why do we add this restriction?

Reply:
Here, we suppose that a node needs to be into at least one hyperedge.

Reply:
I understand, but why is this restriction added?

Reply:
We considered having at minimum to have self-loops hyperedges...

Reply:
We can remove this constrain, and update the doc.
?

Reply:
I am OK with what you proposed - just wanted to understand the constraint.